shoov/modules/custom/shoov_restful/plugins/restful/node/ci_build_status/1.0/ci_build_status__1_0.inc

```diff
@@ -0,0 +1,14 @@
+<?php
+
+$plugin = array(
+  'label' => t('CI-Build status'),
+  'resource' => 'ci-build-status',
+  'name' => 'ci_build_status__1_0',
+  'entity_type' => 'node',
+  'bundle' => 'ci_build',
+  'description' => t('Export the "CI-Builds" status image.'),
+  'class' => 'ShoovCiBuildStatusResource',
+  'authentication_types' => TRUE,
```

Comment:
this gets a bit more tricky here... :)

Let's remove auth type, and:

1) Add `Status token` text field to CI build, which will be populated on hook_node_presave() with a random token
2) Implement `public function access()` that will check in $this->getRequest() that we have the correct `status_token`